test/integration/api/network.bats

```diff
@@ -176,3 +175,15 @@ function teardown() {
 	run docker_swarm network inspect testn
 	[[ "${output}" != *"\"Containers\": {}"* ]]
 }
+
+@test "docker run --net <node>/<network>" {
+	start_docker_with_busybox 2
+	swarm_manage
+
+	docker_swarm network create -d bridge node-1/testn
+
+	docker_swarm run -d --net node-1/testn --name test_container busybox sleep 100
+
+	run docker_swarm network inspect testn
+	[[ "${output}" != *"\"Containers\": {}"* ]]
+}
```

Comment:
Can you also add a non-compose test to handle my previous comment by creating a network with `/` character in it and making sure swarm is able to handle that ?